Gemfile

```diff
@@ -1,12 +1,13 @@
-ruby "2.2.4"
+ruby RUBY_VERSION
```

Comment:
It's not a silly question, it's at the heart of the usability of this feature.

To help us always use the exact version, we can change back to exact match, _and_ add a `.ruby-version` file with that ruby version number to the project. That `.ruby-version` is used automatically by the Travis build's `rvm` step.

And it'll also be used by the passing developer, who's trying to assist this project.

I'll change the file contents!